website/common/script/content/appearance/backgrounds.js

```diff
@@ -8,240 +8,240 @@ import t from '../translation';
 /* eslint-disable camelcase */
 const plannedBackgrounds = {
   backgrounds062014: {
-    beach: { },
-    fairy_ring: { },
-    forest: { },
+    beach: {},
+    fairy_ring: {},
+    forest: {},
```

Comment:
I'm not going to require that you remove these, but I will say that I'd really prefer not to have hundreds of lines of autolinter changes rolled in to an unrelated PR. It makes the whole thing harder to review, which in turn increases the risk of errors slipping through.